src/core/lib/channel/connected_channel.cc

```diff
@@ -888,18 +884,7 @@ ArenaPromise<ServerMetadataHandle> MakeClientTransportCallPromise(
     Transport* transport, CallArgs call_args, NextPromiseFactory) {
   auto spine = GetContext<CallContext>()->MakeCallSpine(std::move(call_args));
   transport->client_transport()->StartCall(CallHandler{spine});
-  return Map(spine->server_trailing_metadata().receiver.Next(),
-             [](NextResult<ServerMetadataHandle> r) {
-               if (r.has_value()) {
-                 auto md = std::move(r.value());
-                 md->Set(GrpcStatusFromWire(), true);
```

Comment:
I don't see anywhere that this moved to.  Are we not setting this attribute anymore?